### PR TITLE
exercise 1

### PR DESCRIPTION
**Pull request by MrTall93**

### CODE REVIEW

File: todo_app/app.py
Comment (JackMead) [inline todo_app/app.py:9]: Combining the POST & GET methods works, but can lead to some surprising behaviour because the POST directly returns the result; e.g. refreshing the page after submitting a new to-do tries to add the same one again. Breaking this out into two separate endpoints (one for GET and one for POST) and having the POST endpoint redirect (using the Flask-provided `redirect` function) to the GET endpoint on success would help with that

**Review by JackMeadDev — APPROVED** [review]
Really nice clean simple solution, I had no issues running this and checking it works. I've left a couple of optional thoughts below, but I'm happy for you to merge this when you're ready

Note that there's no issue with using GitHub but with your cohort we'll typically use GitLab instead to match your workplace, so it would be good to move this over to there (potentially after merging if that's easiest) - there is [some guidance for that here](https://corndelwithsoftwire.github.io/DevOps-Course-FAQs/Tools/gitlab.html)

**Pull request merged**
